Grammar Updates to the General Folder (#1026)
AdguardTeam/AdguardKnowledgeBase: 01.general/06.userscripts/docs.en.md

## Introduction

> First, let's get it clear. What is a userscript? Userscripts (we also call them "extensions" sometimes) are, de-facto, mini-programs written in Javascript. They modify or extend the functionality of one or more websites. Many AdGuard users must already be familiar with such userscripts as AdGuard Assistant, Popup Blocker, and AdGuard Extra.

Author: ```suggestion
First, let's get it clear. What is a userscript? Userscripts (sometimes we also call them "extensions") are, de-facto, mini-programs written in Javascript. They modify or extend the functionality of one or more websites. Many AdGuard users are probably already familiar with such userscripts as AdGuard Assistant, Popup Blocker, and AdGuard Extra.```

<a name="products"></a>

### Supported apps

> Working as a user script manager, AdGuard can significantly broaden website functionality. You can add your custom scripts or manage existing ones in our three products: [AdGuard for Windows](https://kb.adguard.com/en/windows/features/extensions), [AdGuard for Android](https://kb.adguard.com/en/android/overview#extensions-support) and [AdGuard for Mac](https://kb.adguard.com/en/macos/overview#extensions).

Author: ```suggestion
AdGuard can significantly broaden website functionality by acting as a userscript manager. You can add your custom scripts or manage the existing ones in our three products: [AdGuard for Windows](https://kb.adguard.com/en/windows/features/extensions), [AdGuard for Android](https://kb.adguard.com/en/android/overview#extensions-support) and [AdGuard for Mac](https://kb.adguard.com/en/macos/overview#extensions).
```

<a name="scripts"></a>

### Recommended AdGuard scripts

> ### ATTENTION
>

> > These userscripts come directly from AdGuard developers, and we can guarantee that they will be effective and safe. Keep in mind that whenever you download a userscript from an unknown source, you expose yourself to a risk. Some scripts can be harmful to your computer. Always get new scripts only from sources you trust 100%. Furthermore, [some examples](#repo) of popular sites with scripts.

Author: ```suggestion
> These userscripts come directly from the AdGuard developers, and we can guarantee that they are effective and safe. Remember: whenever you download a userscript from an unknown source, you expose yourself to risk, as some scripts may be harmful to your computer. Get new scripts only from the sources you trust 100%. Furthermore, here are [some examples](#repo) of popular sites with scripts.
```
